.github/workflows/publish_and_release.yml

```
@@ -0,0 +1,70 @@
+        python-version: "3.12"
+    
+    - name: Install Poetry
+      run: |
+        curl -sSL "https://install.python-poetry.org" | python3 -
+        echo "$HOME/.local/bin" >> $GITHUB_PATH
+    
+    - name: Configure Git
+      run: |
+        git config --global user.name "GitHub Actions"
+        git config --global user.email "actions@github.com"
+    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)